docs/modules/ROOT/pages/introspector.adoc

```diff
@@ -0,0 +1,92 @@
+[[introspector]]
+= Introspect schema from an existing Neo4j database
```

Comment:
Tied in with my suggestion above, let's give this page a snappy title?

```suggestion
= Introspector
```

Reply:
I'm thinking for SEO, the existing title is much better, that's why I made the link and the page title different.